yt_dlp/extractor/rtllu.py

```diff
@@ -0,0 +1,54 @@
+from .common import InfoExtractor
+
+
+class RTLLuBaseIE(InfoExtractor):
+    _MEDIA_REGEX = {
+        'video': r'<rtl-player\s*poster\s*=\s*\"(?P<thumbnail_url>[\"\w\.://-]+)\"\s*title\s*=\s*(?P<title>[\"\w\.-]+)\s*hls\s*=\s*\"(?P<media_url>[\w\.\:/-]+)\"',
+        'audio': r'<rtl-audioplayer\s*src\s*=\s*\"(?P<media_url>[\w\.\:/-]+)\"',
+    }
+
+    def get_media(self, media_type, webpage, video_id):
+        media_url = self._search_regex(
+            self._MEDIA_REGEX[media_type], webpage, 'media_url', group=('media_url'))
+
+        return media_url
```

Comment:
1. What is the point of this BaseIE? 
2. You are only using `media_type='video'`. What's audio for?
3. Why are you capturing, but discarding thumbnail_url?

This just looks incomplete...

Reply:
Actually, i want to extend this extractor to extract audio in `rtl.lu/radio` and audio/video in `rtl.lu/.../news` but i think it's better if all of these be split to another PR.

Reply:
The problem is, I can't judge whether the code should be structured like this by seeing just this. You could trim down this PR to only what is needed now, but that would be unnecessary work imo. It is better to complete the extractor